cmd/mailchain/internal/http/handlers/openapi.go

```diff
@@ -18,7 +18,7 @@ package handlers
 //nolint: lll
 //nolint: funlen
 func spec() string {
-  return `
+	return `
```

Comment:
Hi @developerfred although this change fixes the `fmt` issues  on `openapi.go` it does not fix the root of the problem. `openapi.go` is generated by running `make openapi`.  Each time time `make openapi` is run the problem will reappear. Can you look at `Makefile` in the `openapi:` section and look to add the `fmt` command towards the end after the file has been created. Thanks :)

Reply:
ok. I will fix this.